src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestEditor/BorrowingRequestEditor.js

```diff
@@ -17,6 +18,20 @@ export class BorrowingRequestEditor extends Component {
     }
   }
 
+  get borrowingRequest() {
+    const request = this.props.location.state || null;
+    if (!request) return null;
```

Comment:
I am a bit confused here: why creating a new borrowing request will have already:
`borrowingRequestPid: request.pid,`?
Isn't it maybe the new request for a new document pid?